style.scss

```diff
@@ -0,0 +1,263 @@
+$font-main: roboto, sans-serif;
+$font-secondary: arial, helvetica, sans-serif;
+$icons: 27px;
+
+$theme-colors: (
+  "primary": red,
+  "secondary": #ccc
+);
+
+body {
+  color: grey;
+}
+
+.mainbody {
+  max-width: 1310px;
+  overflow-x: hidden;
+}
+
+.icons {
+  font-size: $icons;
+}
+
+.navbar {
+  background-color: red;
+
+  * {
+    color: white;
+  }
+
+  img {
+    width: 27%;
+  }
+}
+
+.btn-warning {
+  border-radius: 16px;
+  color: black;
+}
+
+footer {
+  position: relative;
+  bottom: 0;
+}
+
+#footerTitle,
+#footerTitle2 {
+  fill: red;
+  height: 100%;
+  width: 100%;
+}
+
+.logo {
+  fill: white;
+  width: 100%;
+}
+
+.subscribebutton {
+  font-weight: bold;
+  color: black;
+  border-radius: 20px;
+}
+
+.subscribebutton2 {
+  font-weight: bold;
+  color: black;
+  border-radius: 20px;
+  font-size: 10px;
+}
+
+.togglerbutton {
+  width: 5px;
+  height: auto;
+}
+
+.middleNavLinks,
+.footerNavLinks {
+  color: #000;
+  font-weight: bolder;
+}
+
+.footerNavLinks {
+  font-weight: normal;
+}
+
+.middleNavLinks:hover,
+.footerNavLinks:hover {
+  color: red;
+}
+
+.lowerLinks {
+  color: #666;
+}
+
+#lowerLinks1 {
+  color: black;
+}
+
+.footer-links {
+  a {
+    width: 40%;
+  }
+}
+
+i {
+  color: grey;
+}
+
+.nav-link {
+  border-right: 1px  rgba(153, 150, 150, 0.301) solid;
+}
+
+#newsTitle {
+  width: 900px;
+}
+
+#newsTitle2 {
+  height: auto;
+  width: 150px;
+}
+
+// Cards
+
+.card-title,
+.card-text,
+.black {
+  color: black;
+}
+
+.red {
+  color: red;
+}
+
+.card-text-opinion {
+  font-size: 12px;
+}
+
+.cardImgCont {
+  position: relative;
+}
+
+.magImgCont {
+  position: relative;
+}
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-dark`, `position-relative` bootstrap classes. Please check the rest of your styling as well.